### PR TITLE
Add orc reading feedback in log.

### PR DESCRIPTION
**Pull request by Jens-Co**
image

Always useful to know what the ocr reading is on images. incase we have a bad reading we will know what the issue is and maybe fix it.

### CODE REVIEW

**Comment by Jens-Co** [comment]
image

Added author id and timestamp to follow all other logs.

**Comment by Chew** [comment]
imo the link should be a Link button to the Jump URL

**Comment by Jens-Co** [comment]
> imo the link should be a Link button to the Jump URL

Thanks for the suggestion! done!

File: src/main/java/org/geysermc/discordbot/listeners/ErrorAnalyzer.java [hunk 1]
```diff
@@ -131,7 +132,18 @@ public void onMessageReceived(MessageReceivedEvent event) {
...
+                        // Send ocr reading to logs channel.
+                        ServerSettings.getLogChannel(event.getGuild()).sendMessageEmbeds(new EmbedBuilder()
+                                .setTitle("OCR Reading")
+                                .addField("Image link", "[Jump to Image](" + event.getJumpUrl() + ")", true)
+                                .addField("Reading", textFromImage, false)
+                                .setFooter("ID: " + event.getAuthor().getId())
+                                .setTimestamp(Instant.now())
+                                .setColor(BotColors.NEUTRAL.getColor())
+                                .build()).queue();
+                        //
```
Comment (Konicai) [inline src/main/java/org/geysermc/discordbot/listeners/ErrorAnalyzer.java:145]: Empty comment

**Review by Camotoy — APPROVED** [review]

**Review by rtm516 — APPROVED** [review]

**Pull request merged**